ksqldb-common/src/main/java/io/confluent/ksql/util/PushOffsetVector.java

```diff
@@ -164,4 +164,15 @@ public boolean equals(final Object o) {
     final PushOffsetVector that = (PushOffsetVector) o;
     return Objects.equals(offsets.get(), that.offsets.get());
   }
+
+  public static boolean isContinuationTokenEnabled(final Map<String, Object> requestProperties) {
+    final Object continuationTokenEnabled
```

Comment:
Do we also want to check `KSQL_QUERY_PUSH_V2_ENABLED` as well since this is required to enable V2 to begin with?

Reply:
added check